.github/workflows/plugins-tester.yml

```diff
@@ -106,12 +106,6 @@ jobs:
         run: |
             echo "PLUGINS_TESTER__PLUGINS_TO_TEST=${{ github.event.inputs.plugins }}" >> $GITHUB_ENV
             echo "PLUGINS_TESTER__DIFF_THRESHOLD=${{ github.event.inputs.diffThreshold }}" >> $GITHUB_ENV
-      - name: Run wp-env
-        uses: ./.github/workflows/run-wp-env
-        with:
-          PLUGINS: './tests/plugins-tester/plugins/elementor-tests-utils'
-          PORT: '7777' # In order to enable local run side by side with another default server (port 8888)
```

Comment:
Is there a workaround to using this locally if we remove the port here?

Reply:
you can use it locally. It will take 7777 from wp-env.json. (In plugin tester folder). The problem is that you need a linux machine, otherwise screenshots will still have diffs.